app.js

```diff
@@ -53,8 +53,19 @@ function readFileCSV(text, fileName) {
         else if (line[i] === "\"First Name\"") first_name = i;
         else if (line[i] === "\"Last Name\"") last_name = i;
     }
+    if(user_name === first_name || user_name === last_name || first_name === last_name) {
+        swal({
+            title: "Incorrectly Formatted File",
+            text: "Please only drag and drop CSV files given to you from the Blackboard Grade Center. We were not able to detect all of the columns necessary to convert your file.",
+            icon: "warning",
+            buttons: true,
+            dangerMode: true
+        });
+        return;
+    }
     for (var j = 1; j < allTextLines.length - 1; j++) { //skip header line
         line = allTextLines[j].split(',');
+        if (line[user_name].substr(0,8) === "\"clicker") continue; //skip over clicker users

```

Comment:
Can we also include Preview Users in the skip list?

Reply:
updated in my branch, we just need to test